kubernetes-model-generator/kubernetes-model-admissionregistration/src/generated/java/io/fabric8/kubernetes/api/model/admissionregistration/v1beta1/ValidatingWebhook.java

```diff
@@ -73,17 +72,14 @@ public class ValidatingWebhook implements Editable<ValidatingWebhookBuilder> , K
     private WebhookClientConfig clientConfig;
     @JsonProperty("failurePolicy")
     private String failurePolicy;
-    @JsonProperty("matchConditions")
-    @JsonInclude(JsonInclude.Include.NON_EMPTY)
-    private List<MatchCondition> matchConditions = new ArrayList<>();
```

Comment:
Why are we removing this field?

I see a field named `matchCondition` in [admissionregistration/v1beta1/types.go](https://github.com/kubernetes/api/blob/b689d905290f2d04d983e10e3705bb6efe94e046/admissionregistration/v1beta1/types.go#L901)

Reply:
This is the v1beta1 spec.
I cannot find the one that's exactly the same as the one we have in our repo in any of the available OpenAPI JSONS.
This was probably part of one of the K8s versions at some point.

Since this a is an unstable version, I think it's OK to drop field.
Otherwise we might need to add the spec for an older k8s version (we already have 1.17, 1.21, 1.26, and 1.30). I don't really think it's worth the effort to support something which is deprecated since we're shifting the major version of the Kubernetes Client.